draft-packages/badge/KaizenDraft/Badge/styles.module.scss

```diff
@@ -24,7 +25,7 @@ $small: $ca-grid;
 }
 
 .default {
-  background-color: $kz-color-wisteria-200;
+  background-color: $dt-color-background-default;
   color: $kz-color-wisteria-800;
```

Comment:
Same here, could change to `$kz-var-color-wisteria-800`, import `color-vars` and remove the unused `color` import.

Reply:
I was thinking I only wanted to make the required change (the things that aren't 1-1 token swaps) but I forgot that without the CSS variable tokens everywhere the tokens that have the same name but different values won't swap 🤦 

Will follow that up. Thanks!